### DISCUSSION TITLE
Support for SvelteKit 2

### DISCUSSION THREAD

**Discussion by irt-msummers** [etherCorps/SK-Redis-SessionManager#8] [General]
I know it just came out about a week ago, but do you know when you might have support for SvelteKit 2?
Thanks

**Comment by theetherGit** [comment]
Good question, I haven't tested it but i can say it might work with sveltekit 2 but i have to update the dependencies.

**Comment by irt-msummers** [comment]
Thank you

**Reply by theetherGit** [reply]
it works with v2?

**Comment by irt-msummers** [comment]
I did npx svelte-migrate sveltekit-2 on my project and installed yours with a --force and it seems to work
